src/main/resources/com/google/api/codegen/nodejs/sample.snip

```diff
@@ -62,10 +62,14 @@
     @end
     @if class.hasRequestBody
       resource: {
-        @if class.methodVerb == "PATCH"
-          // TODO: Add desired properties of the request body. Only added properties
+        @switch class.methodVerb
+        @case "PATCH"
+          // TODO: Add desired properties of the request body. Only these properties
```

Comment:
Should we change these to "Assign values to desired properties of..." to be have wording more consistent with the other languages'?